pinc/project_trans.inc

```diff
@@ -22,7 +22,7 @@ function project_transition($projectid, $new_state, $who, $extras = [])
     $transition_str = "$projectid: $current_state -> $new_state";
     if ($testing) {
         $indent = '    ';
-        if (false) {
+        if (false) { /** @phpstan-ignore-line */
```

Comment:
I'd remove this block (and the commented-out echo on line 35).

Reply:
Done.